src/dragonball/Cargo.toml

```diff
@@ -61,3 +61,4 @@ virtio-net = ["dbs-virtio-devices/virtio-net", "virtio-queue"]
 virtio-fs = ["dbs-virtio-devices/virtio-fs", "virtio-queue", "atomic-guest-memory"]
 virtio-mem = ["dbs-virtio-devices/virtio-mem", "virtio-queue", "atomic-guest-memory"]
 virtio-balloon = ["dbs-virtio-devices/virtio-balloon", "virtio-queue"]
+vhost-net = ["virtio-net", "dbs-virtio-devices/vhost-net"]
```

Comment:
Why vhost-net depends on virtio-net

Reply:
Fixed. Dragonball's vhost-net feature not depends on virtio-net feature now.